### PR TITLE
Add bookmark locator filter to QField's search bar

### PR DESCRIPTION
**Pull request by nirvn**
Part 2 of the bookmark support implementation in QField:
image

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 024114da030251e2d9bccc2d6d9fd5b939745ed1:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-664aed9f62ce0762f020413c4a6ee4604ca169c3-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-664aed9f62ce0762f020413c4a6ee4604ca169c3-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-664aed9f62ce0762f020413c4a6ee4604ca169c3-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-664aed9f62ce0762f020413c4a6ee4604ca169c3-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 41e7f5e7929ca37c64aed2daa9ff278d57dd6000:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-89fe2835e2a36f0124d5d674ee96a6e81a6e2791-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-89fe2835e2a36f0124d5d674ee96a6e81a6e2791-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-89fe2835e2a36f0124d5d674ee96a6e81a6e2791-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-89fe2835e2a36f0124d5d674ee96a6e81a6e2791-x86.apk)

File: src/app/main.cpp [hunk 1]
```diff
@@ -195,7 +197,6 @@ int main( int argc, char **argv )
 #endif
 #endif
 
-  qInstallMessageHandler( qfMessageHandler );
```
Comment (m-kuhn) [inline src/app/main.cpp:198]: What was wrong here?

**Comment by m-kuhn** [comment]
Looks very good to me apart from the one question, thanks !

File: src/app/main.cpp (hunk 1 above)
Comment (nirvn) [reply src/app/main.cpp:198]: I just realized you're using this for sentry too. What happens here is that on linux builds, we're neither ANDROID nor SENTRY, and yet we register the message handler, which as a result eats all messages and outputs nothing.

Comment (nirvn) [reply src/app/main.cpp:198]: @m-kuhn , pushed better fix.

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 967682f4ec18de550d1309bbfb055fd6ff0a0089:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-a621ceba589f84e50131d971ec28c9c5e5021944-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-a621ceba589f84e50131d971ec28c9c5e5021944-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-a621ceba589f84e50131d971ec28c9c5e5021944-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-a621ceba589f84e50131d971ec28c9c5e5021944-x86.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 24db5c5ce03b675b37caefa7fef9281498e49079:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-d6c904482443ffa5a0967abe6d19e1886a032e10-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-d6c904482443ffa5a0967abe6d19e1886a032e10-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-d6c904482443ffa5a0967abe6d19e1886a032e10-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2396-d6c904482443ffa5a0967abe6d19e1886a032e10-x86.apk)

**Review by m-kuhn — APPROVED** [review]

**Pull request merged**